pkg/controller/namespace/deletion/namespaced_resources_deleter.go

```diff
@@ -304,12 +306,13 @@ func (d *namespacedResourcesDeleter) finalizeNamespace(namespace *v1.Namespace)
 // deleteCollection is a helper function that will delete the collection of resources
 // it returns true if the operation was supported on the server.
 // it returns an error if the operation was supported on the server but was unable to complete.
-func (d *namespacedResourcesDeleter) deleteCollection(gvr schema.GroupVersionResource, namespace string) (bool, error) {
-	klog.V(5).Infof("namespace controller - deleteCollection - namespace: %s, gvr: %v", namespace, gvr)
+func (d *namespacedResourcesDeleter) deleteCollection(ctx context.Context, gvr schema.GroupVersionResource, namespace string) (bool, error) {
+	logger := klog.FromContext(ctx)
+	logger.V(5).Info("Namespace controller - deleteCollection", "namespace", namespace, "resource", gvr)
```

Comment:
```suggestion
	logger.V(5).Info("Namespace controller - deleteCollection -", "namespace", namespace, "groupVersionResource", gvr)
```

Reply:
why do we need -?

Reply:
Yes, let's not add that. It's similar to "end with a period".

The [guidelines](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/migration-to-structured-logging.md#remove-string-formatting-from-log-message) for log messages are

* Start from a capital letter.
* Do not end the message with a period
* Use active voice. Use complete sentences when there is an acting subject ("A could not do B") or omit the subject if the subject would be the program itself ("Could not do B").
* Use past tense ("Could not delete B" instead of "Cannot delete B")
* When referring to an object, state what type of object it is. ("Deleted pod" instead of "Deleted")

Reply:
I find `groupVersionResource` too verbose: the item is a "resource", identified by a group/version/resource.